agenta-web/src/components/pages/observability/drawer/TraceContent.tsx

```diff
@@ -260,21 +262,25 @@ const TraceContent = ({activeTrace}: TraceContentProps) => {
                             {activeTrace.node.name}
                         </Typography.Text>
 
-                        {/* <Space>
-                            {!activeTrace.parent && activeTrace.refs?.application?.id && (
+                        <Space>
+                            {/* {!activeTrace.parent && activeTrace.refs?.application?.id && (
```

Comment:
looks like some cleanup is needed. or is there any reason to keep this commented part?